src/components/SnackChat/SnackChat.js

```diff
@@ -118,26 +166,37 @@ class SnackChat extends Component {
 
     // Filter
     this.ctx.save();
-    this.ctx.rotate(body.shoulders.angle);
+    this.ctx.rotate(this.averageBodyPosition.shoulders.angle);
     this.ctx.drawImage(
       this.filter,
-      body.shoulders.right.x -
-        body.shoulders.span * 1.5 +
-        body.shoulders.span * body.shoulders.angle,
-      body.shoulders.right.y - body.shoulders.span * 1.5,
-      body.shoulders.span * 4,
-      body.shoulders.span * 4
+      this.averageBodyPosition.shoulders.rightX -
```

Comment:
might be easier to at the top of the function reassign `const shoulders = this.averageBodyPosition.shoulders`
and same for ears, just these variables is quite long - which is good for readability but i think just shoulders would be fine